src/Components/News/NewsList.css

```diff
@@ -4,4 +4,14 @@
     grid-row-gap: 2em;
     grid-template-columns: repeat(auto-fit, minmax(18.75rem, 1fr));
     margin-top: 7em;
-}
\ No newline at end of file
+}
+
+.news-list__image:after {  
```

Comment:
what is adding this?

Reply:
![image](https://user-images.githubusercontent.com/37185433/52234657-ca39ea80-2887-11e9-8548-4b707b7dd2f2.png)

hides broken img icon